var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -176,10 +180,16 @@ def common_config_args(self):
             '-optimized-qmake',
             '-no-openvg',
             '-no-pch',
-            # NIS is deprecated in more recent glibc
-            '-no-nis'
         ]
 
+        if '@:5.7.0' in self.spec:
+            config_args.extend([
+                # NIS is deprecated in more recent glibc,
+                # but qt-5.7.1 does not recognize this option
+                '-no-nis',
+            ])
+
+
```

Comment:
These 2 blank lines are causing the flake8 tests to fail. You should delete one of them.